examples/docker/fastapi-ai-helper/README.md

```diff
@@ -0,0 +1,76 @@
+# About this application
+
+This is an LLM-based API with a focus on helping you answer questions about orders. 🚀
+
+## How it works
+
+This API uses the  Haystack (https://haystack.deepset.ai/overview/intro) framework to build a 
+Retrieval Augmented Generation (RAG) pipeline that can answer questions about orders. 
+The pipeline is initialized with a document store that contains information about orders.
+The pipeline uses the document store to answer questions about orders.
+
+The API is built with FastAPI (https://fastapi.tiangolo.com/), a modern, fast (high-performance),
+web framework for building APIs with Python 3.6+ based on standard Python type hints.
+
+It is deployed on  Ploomber Cloud (https://ploomber.io/), a platform for building and deploying
+Python data and AI-based apps.
+
+## Sample data
```

Comment:
any reason why we have a single invoice? it'd be better to have at least a few orders to make this a bit more realistic

Reply:
I think adding a few orders from a few customers would be a lot more realistic. and then the chatbot could also validate that the order belongs to the user

e.g., if I'm user A, i should only be allowed to get info on orders that user A made

Reply:
nvm. I noticed that there are a few orders in the dataset.

but the second point still holds, there should be some kind of authentication so I can only retrieve info about my orders (e.g. you can ask for the customer's name or ID at the beginning of the chat)

Reply:
> any reason why we have a single invoice? it'd be better to have at least a few orders to make this a bit more realistic

Resolved here

https://github.com/ploomber/doc/pull/90/commits/4b9804eb911c9596c981d48fa567687f9a37c336

> I think adding a few orders from a few customers would be a lot more realistic. and then the chatbot could also validate that the order belongs to the user
> 
> e.g., if I'm user A, i should only be allowed to get info on orders that user A made

This was much more challenging to implement with the LLM and prompt alone. I would need to implement a data entry functionality through something like a UI that is then connected to the API, but via prompting and the api alone the results were bad enough I don't want to add that as a question. Kept original prompt